src/Core/System/Snippet/SnippetValidator.php

```diff
@@ -48,13 +47,23 @@ public function validate(): array
         return $this->findMissingSnippets($snippetFileMappings, $availableISOs);
     }
 
-    protected function getAllFiles(): SnippetFileCollection
+    /**
+     * @param array<int, string> $bundles
+     */
+    protected function getAllFiles(array $bundles = []): SnippetFileCollection
```

Comment:
this is still a break. if someone is extending this class and is overwriting this method, it will break.